ansible/roles/cartodb.resque/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+ - name: Ensure pid file dir exists
+   command: mkdir -p {{ cartodb_resque_pid_dir }}
```

Comment:
Just noticed that Ansible 2.0 has a nice warning about this:
```
TASK [cartodb.resque : Ensure pid file dir exists] *****************************
changed: [cartodb]
 [WARNING]: Consider using file module with state=directory rather than running
mkdir
```